dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard.feature

```diff
@@ -175,23 +175,6 @@ Feature: Using the teacher dashboard
     And I press the first ".project_remix" element to load a new page
     And I wait for the page to fully load
 
-    # Create a playlab project level and generate a thumbnail.
```

Comment:
Removed this test because we no longer support share/remix for Play Lab project template levels, nor adding to projects gallery for free-play levels.